src/hermes/commands/harvest.py

```diff
@@ -29,25 +37,14 @@ def harvest_cff(click_ctx: click.Context, ctx: HermesContext):
         return 1
     else:
```

Comment:
`else` is not required as the `if` branch returns. Reduced indentation == reduced complexity (at least if you are a programming metrics tools)

Reply:
Good point. Fixed in 2e709b601cae318ec72c93a9b78610a1db3b5cd1.